dhis-2/dhis-test-integration/src/test/resources/tracker/tracker_metadata_with_program_rules.json

```diff
@@ -134,6 +135,9 @@
       "lastUpdatedBy": {
         "id": "M5zQapPyTZI"
       },
+      "createdBy": {
+        "id": "M5zQapPyTZI"
```

Comment:
AFAIK created/update users and dates will be ignored as its set by some base crud store. Adding these in fixtures is thus confusing 😬

Reply:
Fixed